### PR TITLE
Individual reports

### PR DESCRIPTION
**Pull request by petmetic**
Individual hours report

### CODE REVIEW

File: src/web/templates/web/myreport.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+{% block title %}{% translate "Detail Breakdow of Hours" %}{% endblock %}
+
+{% block content %}
+    <div class="container text-left">
+        <div class="p-4">
+            <div class="row p-1">
+                <h4 class="col-sm-8 fw-light uppercase"><span
+                        class="fw-lighter uppercase">{% translate "Therapist" %}:</span> {{ therapist.first_name }}</h4>
+<div>
+                <form action="." method="get" class='row  align-items-end'>
```
Comment (jurecuhalev) [inline src/web/templates/web/myreport.html:12]: Be consistent: either use `"` or `'` not both in the same html document

File: src/web/templates/web/reports.html [hunk 2]
```diff
@@ -4,27 +4,26 @@
 
 {% block content %}
     <h4 class="p-4 fw-lighter uppercase">{% translate "Reports" %}</h4>
-    <div class="d-flex justify-content-between align-items-center">
-        <form action="/reports" method="get" class='row'>
-            <div class="col-md-4 fw-light">
+        <form action="/reports" method="get" class='row align-items-end'>
```
Comment (jurecuhalev) [inline src/web/templates/web/reports.html:7]: html consistency

File: src/web/views.py [hunk 3]
```diff
@@ -267,6 +267,44 @@ def report_hours_detail(request, pk: int):
...
+        Massage.objects.filter(therapist=therapist)
+        .filter(status="approved")
+        .filter(start__range=(start_day, end_day))
+        .exclude(customer__name="prostovoljec")
+        .order_by("start")
+    )
+
+    amount = 0
+    for massage in massages:
+        amount += massage.service.payout
```
Comment (jurecuhalev) [inline src/web/views.py:291]: You can do this with Django's Sum - https://docs.djangoproject.com/en/4.2/topics/db/aggregation/#following-relationships-backwards
Improve this code in followup PR.

**Review by jurecuhalev — APPROVED** [review]

File: src/web/views.py (hunk 3 above)
Comment (petmetic) [reply src/web/views.py:291]: [improve code for adding payout amount#145](https://github.com/petmetic/therapy-tracker/issues/145)

File: src/web/templates/web/reports.html [hunk 4]
```diff
@@ -4,27 +4,26 @@
 
 {% block content %}
     <h4 class="p-4 fw-lighter uppercase">{% translate "Reports" %}</h4>
-    <div class="d-flex justify-content-between align-items-center">
-        <form action="/reports" method="get" class='row'>
-            <div class="col-md-4 fw-light">
+        <form action="/reports" method="get" class="row align-items-end">
```
Comment (jurecuhalev) [inline src/web/templates/web/reports.html:7]: use {% url %}

File: src/web/templates/web/myreport.html [hunk 5]
```diff
@@ -0,0 +1,67 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% block title %}{% translate "Detail Breakdow of Hours" %}{% endblock %}
```
Comment (jurecuhalev) [inline src/web/templates/web/myreport.html:3]: typo

File: src/web/templates/web/myreport.html [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+                    <td colspan="4" class="col fw-bold uppercase">{% translate "To be paid in total to therapist" %}:
+                    </td>
+                    <td colspan="1" class="col-sm fw-bold">{{ amount }} €</td>
+                </tr>
+                </tfoot>
+            </table>
+        </div>
+        {#        <div class="col-12 px-4">#}
+        {#            <a class="btn btn-body" href="#">{% translate "Download" %}</a>#}
+        {#        </div>#}
```
Comment (jurecuhalev) [inline src/web/templates/web/myreport.html:65]: remove dead code

**Review by jurecuhalev — APPROVED** [review]

**Pull request merged**